### PR TITLE
Small correction toolchains on Stanage

### PR DESCRIPTION
**Pull request by Carldkennedy**

### CODE REVIEW

**Review by nikishe — COMMENTED** [review]
LGTM

**Comment by willfurnass** [comment]
`fosscuda` not installed at present? 

Suggest we only install more in the way of older CUDA tooling if we really need to.  Folks will get best perf if they adopt CUDA >= 11 to make best use of the A100s and >= 11.8 to make best use of the H100s.

**Comment by Carldkennedy** [comment]
> `fosscuda` not installed at present?
> 
> Suggest we only install more in the way of older CUDA tooling if we really need to. Folks will get best perf if they adopt CUDA >= 11 to make best use of the A100s and >= 11.8 to make best use of the H100s.

You're right @willfurnass, not installed. I'll remove those sections.

File: stanage/software/development/toolchains.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ was installed using `EasyBuild <https://easybuild.readthedocs.io/>`__,
 which is a tool for building and installing software on HPC systems.
 
 EasyBuild groups core sets of dependencies as what it calls *toolchains*.
-The three supported toolchains on Stanage are:
+The two supported toolchains on Stanage are:
```
Comment (jkwmoore) [inline stanage/software/development/toolchains.rst:11]: Should this really be "The two main toolchains 'with support|available' on Stanage are" ? Is this support from us or just availability on the cluster?

If we mean availability we should say that instead I think.

**Review by jkwmoore — CHANGES_REQUESTED** [review]
LGTM - we may just want to decide on whether the suggested amendment is needed.

**Review by nikishe — APPROVED** [review]
LGTM (Post jame's suggestion)

File: stanage/software/development/toolchains.rst (hunk 1 above)
Comment (Carldkennedy) [reply stanage/software/development/toolchains.rst:11]: The two main toolchains available with support on Stanage are: ?

Comment (jkwmoore) [reply stanage/software/development/toolchains.rst:11]: Well I suppose it depends mainly on whether you fellas want to explicitly say you will support those toolchains, if yes, how? The latest 2 years worth etc...

I'm not averse to saying their available but at some point we want to explicitly declare our intentions in terms of EB toolchain support formally.

This page seems like the sensible place?

Comment (willfurnass) [reply stanage/software/development/toolchains.rst:11]: Suggest getting this merged and revisiting this conversation via separate Issue/PR.

**Pull request merged**